scripts/pr-deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+    echo "Not a PR. Skipping surge deployment"
+    exit 0
+fi
+
+ember build production
```

Comment:
This is already run

Reply:
Ember Build Production is needed for succesfull surge deployment. I am thinking to put deploy preview in another job other than tests.